src/app/app.module.ts

```diff
@@ -1,13 +1,14 @@
 import { BrowserModule } from '@angular/platform-browser';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
 import { NgModule } from '@angular/core';
+import { HttpModule, Http } from '@angular/http';
```

Comment:
You should consider using `HttpClient` instead of `Http` since it is deprecated.

Reply:
Okay. Will update for HttpClient.